bson/src/main/scala-2.13-/types.scala

```diff
@@ -919,6 +919,7 @@ object BSONElementSet {
   implicit def apply(set: Traversable[BSONElement]): BSONElementSet =
     new BSONDocument(set.map(Success(_)).toStream)
 
+  private[reactivemongo] val typePrefixByteSize = 2
```

Comment:
As far as I can see/remind the [type code is written](https://github.com/ReactiveMongo/ReactiveMongo/blob/master/bson/src/main/scala/buffer/DefaultBufferHandler.scala#L81) as a single byte (or [there](https://github.com/mongodb/mongo-go-driver/blob/03f71cffc6c1b6c1de8b5c6d988d04916f4fbf7d/bson/bsonrw/value_reader.go#L684))

Reply:
Ah yes you're right ! In fact this counts the type code and the null character from the key I guess. I'll change the name